src/getting-started.md

```diff
@@ -33,71 +33,65 @@ instead.
 
 ### Cards
 
-A question and answer pair is called a 'card'. This is based on a paper
-flashcard with a question on one side and the answer on the back. In
-Anki a card doesn’t actually look like a physical card, and when you
-show the answer the question remains visible by default. For example, if
-you’re studying basic chemistry, you might see a question like:
+A question and answer pair is called a _card_. It's similar to a paper
+flashcard with a question on front and answer on the back. However, in
+Anki, a card doesn't actually look like a physical card, and when you
+look at the answer the question remains visible by default. For example, if
```

Comment:
```suggestion
flashcard with a question on the front and an answer on the back. However, in
Anki, a card doesn't look like a physical card, and when you
show the answer the question remains visible by default. For example, if
```

- Edited for grammar.
- Using both "however" and "actually" was too much. 
- Using both "look like" and "look at" was too much. "Show" is a good word to use there, since that's what the button says. "Reveal" is another possibility.